pkg/volume/glusterfs/glusterfs.go

```diff
@@ -135,6 +135,10 @@ type glusterfsBuilder struct {
 
 var _ volume.Builder = &glusterfsBuilder{}
 
+func (_ *glusterfsBuilder) SupportsOwnershipManagement() bool {
```

Comment:
@rootfs Meaning the client could do the `chmod` and `chown` calls without escalating privileges on the server?